### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by yoava333**
Just a few small fixes / additions that gathered in my local repo:
* added winafl-plot.py a winafl alternative for afl-plot & fixed compatibility issues with python3 in winafl-whatsup.py
* fixed a minor memory leak
* added console color support for afl-tmin and afl-showmap

### CODE REVIEW

File: winafl-plot.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    # get banner
+    stats_path = os.path.join(args.sync_dir, 'fuzzer_stats')
+    stats = parse_fuzzer_stats(stats_path)
+    banner = stats['afl_banner']
+
+    # format date in a compatible manner
+    date = time.strftime('%a %b %d %H:%M:%S DST %Y', time.localtime())
+
+    # write html file
+    index_html = os.path.join(args.out_dir, 'index.html')
```
Comment (ivanfratric) [inline winafl-plot.py]: Create out_dir if it doesn't already exist

File: winafl-plot.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+    tmp_file = tempfile.NamedTemporaryFile(delete=False)
+    tmp_file.write(GNUPLOT_CMDS.format(outdir=args.out_dir,
+                                       fuzzer_dir=args.sync_dir).encode('utf-8'))
+    tmp_file.flush()
+
+    # run gnuplot
+    try:
+        subprocess.check_output(['gnuplot', '-c', tmp_file.name])
+    except subprocess.CalledProcessError as e:
```
Comment (ivanfratric) [inline winafl-plot.py]: Possibly other errors should be caught here, for example this is what I get without gnuplot in PATH

Traceback (most recent call last):
  File "winafl-plot.py", line 132, in <module>
    main()
  File "winafl-plot.py", line 121, in main
    subprocess.check_output(['gnuplot', '-c', tmp_file.name])
  File "C:\Python27\lib\subprocess.py", line 567, in check_output
    process = Popen(stdout=PIPE, *popenargs, **kwargs)
  File "C:\Python27\lib\subprocess.py", line 711, in __init__
    errread, errwrite)
  File "C:\Python27\lib\subprocess.py", line 959, in _execute_child
    startupinfo)
WindowsError: [Error 2] The system cannot find the file specified

Something more user-friendly would be nice (e.g. telling the user to install gnuplot and add it to PATH)

**Comment by yoava333** [comment]
Fixed comments and pushed force the changes

**Comment by ivanfratric** [comment]
Cool, thanks!

**Pull request merged**